rust-runtime/aws-smithy-legacy-http/Cargo.toml

```diff
@@ -0,0 +1,52 @@
+[package]
+name = "aws-smithy-legacy-http"
+version = "0.62.5"
+authors = [
+    "AWS Rust SDK Team <aws-sdk-rust@amazon.com>",
+    "Russell Cohen <rcoh@amazon.com>",
+]
+description = "Smithy HTTP-0x logic for smithy-rs."
+edition = "2021"
+license = "Apache-2.0"
+repository = "https://github.com/smithy-lang/smithy-rs"
+
+[features]
+event-stream = ["aws-smithy-eventstream"]
+rt-tokio = ["aws-smithy-types/rt-tokio"]
+
+[dependencies]
+aws-smithy-eventstream = { path = "../aws-smithy-eventstream", optional = true }
+aws-smithy-runtime-api = { path = "../aws-smithy-runtime-api", features = ["client", "http-02x"] }
+aws-smithy-types = { path = "../aws-smithy-types", features = ["byte-stream-poll-next", "http-body-0-4-x"] }
+bytes = "1.10.0"
+bytes-utils = "0.1"
+# TODO(hyper1) - Complete the breaking changes by updating to http 1.x ecosystem fully in this crate. Also remove hyper 0.14 from dev
+http-02x = { package = "http", version = "0.2.9" }
+http-1x = { package = "http", version = "1" }
```

Comment:
we don't need a lot of this stuff right? I guess I wonder if we should prune out the parts of aws-smithy-http-legacy we don't need. maybe its OK.